### ISSUE TITLE
Is scannetv2 validation set included in the training process ?

### ISSUE THREAD

**Issue by gaoxiangjun** [yyfz/Pi3#25]
;-) Is scannetv2 validation set included in the training process ?

**Comment by ZhouTimeMachine** [comment]
Hi, we only use trainset of ScanNet, and scannetv2 validation set is not included in our training data.
